messaging/methods/messages/createMessage.md

```diff
@@ -32,6 +32,7 @@ When sending a group message to an invalid phone number, you may receive extrane
 | media*        | No (required if no text is provided)       | `array`                                                        | A list of URLs to include as media attachments as part of the message. If this field is included, the message will be sent as MMS no matter the number of recipients.<br>Refer to the [What MMS message size limits must be adhered to?](https://support.bandwidth.com/hc/en-us/articles/360014235473-What-MMS-message-size-limits-must-be-adhered-to-) article to learn more about file size limitations and best practices. |
 | tag           | No        | `string`                                                       | Any string which will be included in the callback events of the message. (max 1024 Chars)                                                                                                                                                                                                                                                                                                                                     |
 | priority      | No        | `string` | The message's priority, currently for toll-free or short code SMS only. Accepted values are `"default"` or `"high"`. Messages with a priority value of `"high"` are given preference over your other traffic. If not set in the request body, Bandwidth will use the `"default"` value. |
+| expiration    | No        | `string` | The message's expiration date and time, currently for toll-free or short code SMS only. Accepted values must be in RFC-3339 format. Messages received after the expiration date and time will not be sent. Ex: `2021-10-02T15:00:00Z`|
```

Comment:
if a message with an expiration in the future is accepted by the API and queued, and then the expiration lapses before the message is sent, we drop the message and send a [message-failed callback with error code 4360](https://dev.bandwidth.com/messaging/errors/codes.html#client-errors). is that worth calling out here?